data/tabular/BBBP/transform.py

```diff
@@ -0,0 +1,25 @@
+import pandas as pd
+
+
+def load_dataset():
+    BBBP = pd.read_csv(

```

Comment:
non-pythonic (all caps is typically for globals) but otherwise looks good to me!